examples/updatecli.generic/maven.yaml

```diff
@@ -1,81 +1,30 @@
 ---
```

Comment:
@dduportal What do you think to deprecate the directory examples in favor of e2e which is tested with the latest updatecli version

Reply:
Look good to me. Incoming PR for this (so I'll add all the others yaml from example to e2e).